integrations/github/highlights.scm

```diff
@@ -0,0 +1,214 @@
+; ; Highlighting for TLA+ files on GitHub
+; ; Capture names found here:
+; ; https://github.com/tree-sitter/tree-sitter/blob/f5d1c0b8609f8697861eab352ead44916c068c74/cli/src/highlight.rs#L150-L171
+
+; TLA+ Keywords
+[
+  "ACTION"
+  "ASSUME"
+  "ASSUMPTION"
+  "AXIOM"
+  "BY"
+  "CASE"
+  "CHOOSE"
+  "CONSTANT"
+  "CONSTANTS"
+  "COROLLARY"
+  "DEF"
+  "DEFINE"
+  "DEFS"
+  "DOMAIN"
+  "ELSE"
+  "ENABLED"
+  "EXCEPT"
+  "EXTENDS"
+  "HAVE"
+  "HIDE"
+  "IF"
+  "IN"
+  "INSTANCE"
+  "LAMBDA"
+  "LEMMA"
+  "LET"
+  "LOCAL"
+  "MODULE"
+  "NEW"
+  "OBVIOUS"
+  "OMITTED"
+  "ONLY"
+  "OTHER"
+  "PICK"
+  "PROOF"
+  "PROPOSITION"
+  "PROVE"
+  "QED"
+  "RECURSIVE"
+  "SF_"
+  "STATE"
+  "SUBSET"
+  "SUFFICES"
+  "TAKE"
+  "TEMPORAL"
+  "THEN"
+  "THEOREM"
+  "UNCHANGED"
+  "UNION"
+  "USE"
+  "VARIABLE"
+  "VARIABLES"
+  "WF_"
+  "WITH"
+  "WITNESS"
+  (address)
+  (all_map_to)
+  (assign)
+  (case_arrow)
+  (case_box)
+  (def_eq)
+  (exists)
+  (forall)
+  (gets)
+  (label_as)
+  (maps_to)
+  (set_in)
+  (temporal_exists)
+  (temporal_forall)
+] @keyword
+
+;  PlusCal keywords
+[
+  "assert"
+  "await"
+  "begin"
+  "call"
+  "define"
+  "either"
+  "else" 
+  "elsif"
+  "end"
+  "fair"
+  "goto"
+  "if" 
+  "macro"
+  "or"
+  "print"
+  "procedure"
+  "process"
+  "return"
+  "skip"
+  "variable"
+  "variables"
+  "when"
+  "with"
+  "then" 
+  (pcal_algorithm_start)
+  (pcal_end_either)
+  (pcal_end_if)
+  (pcal_process ("="))
+  (pcal_with ("="))
+] @keyword
```

Comment:
I forgot to add the "algorithm" token highlighting. Can we include it in this pr?
![image](https://user-images.githubusercontent.com/15569000/152230996-41f73f18-4f99-4575-834e-378220c6ef55.png)